src/__tests__/AtomId.test.ts

```diff
@@ -0,0 +1,41 @@
+import AtomId from '../AtomId';
+import IndexMap from '../IndexMap';
+import getAtomID from '../utils/factories/AtomId';
+
+describe('Atom', () => {
+  describe('toString', () => {
+    it('should print according to defined format', () => {
+      const id = getAtomID();
+      expect(id.toString()).toEqual(`S${id.site}@T${id.timestamp}`);
```

Comment:
Here you're rewriting the implementation. It's best to be explicit here, specifiying the atom ID and timestamp, and then checking the expected string (e.g., "S3@T10").

If you want to check multiple values, write a parameterized test with `each`.